konan/platforms/zephyr/stm32f4_disco

```diff
@@ -34,3 +34,9 @@ dependencies.osx-zephyr_stm32f4_disco = \
     gcc-arm-none-eabi-7-2017-q4-major-mac \
     clang-llvm-5.0.0-darwin-macos \
     target-sysroot-2-wasm
+
+targetToolchain.mingw-zephyr_stm32f4_disco = gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi
```

Comment:
Is it from https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads?